Date: Sep 10, 2018 10:20 PM

From: d82599ca4b4764af42189b5a0d482d376215aacc  Sep 26, 2018 8:45 PM

Subject: [PATCH] RDKB-18052: Use OAUTH to replace POTD

        Turn on curl and openssl in php and add patch file to remove md4 from openssl.

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: Kirk Davis <kirk_davis@cable.comcast.com>
Patch recreated for Yocto version Kirkston by Arjun <arjun_daasuramdass@comcast.com>

---
--- a/ext/openssl/openssl.c	2022-11-16 15:14:04.704176056 +0000
+++ b/ext/openssl/openssl.c	2022-11-16 15:36:06.258859475 +0000
@@ -1121,9 +1121,11 @@
 		case OPENSSL_ALGO_MD5:
 			mdtype = (EVP_MD *) EVP_md5();
 			break;
+#ifdef HAVE_OPENSSL_MD4_H
 		case OPENSSL_ALGO_MD4:
 			mdtype = (EVP_MD *) EVP_md4();
 			break;
+#endif
 #ifdef HAVE_OPENSSL_MD2_H
 		case OPENSSL_ALGO_MD2:
 			mdtype = (EVP_MD *) EVP_md2();
@@ -1146,9 +1148,11 @@
 		case OPENSSL_ALGO_SHA512:
 			mdtype = (EVP_MD *) EVP_sha512();
 			break;
+#ifdef HAVE_OPENSSL_RMD160_H
 		case OPENSSL_ALGO_RMD160:
 			mdtype = (EVP_MD *) EVP_ripemd160();
 			break;
+#endif
 		default:
 			return NULL;
 			break;
